client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -324,10 +334,23 @@ const Progress: React.FC<IProgressProps> = ({
             )
           }
           if (!value) return null
+          if (!showDiscrepanciesReview)
+            return (
+              <>
+                <Icon icon="flag" intent="danger" /> {value.toLocaleString()}
+              </>
```

Comment:
Delete the wrapping brackets?

Reply:
Ah yeah it looks confusing but they are actually needed as they wrap the Icon and a string

Reply:
Oh absolutely right, I missed the string in the right half of the line